clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -59,6 +59,7 @@
 public final class ConsumerCoordinator extends AbstractCoordinator {
 
     private static final Logger log = LoggerFactory.getLogger(ConsumerCoordinator.class);
+    private static final Set<String> INTERNAL_TOPICS = new HashSet<String>();
```

Comment:
We should not modify the set after assigning it to a final field. We should assign it once in the `static` block. This ensures that other threads will see the value correctly. It may also be a good idea to wrap it in `Collections.unmodifiableSet` to ensure accidental mutation doesn't happen.